src/index.css

```diff
@@ -1,5 +1,12 @@
 body {
-  margin: 0 auto;
   box-sizing: border-box;
   font-family: 'Courier New', Courier, monospace;
 }
+
+.box-container {
+  margin: 0 50px;
+}
+.box {
+  max-width: fit-content;
+  border: 0.4px solid rgb(7, 10, 57);
+}
```

Comment:
- [ ] Kinldy make sure you stick to the provided wireframe as suggested in the project requirements.
HINT: You may consider using a `table element` to display your item.
### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/225506074-06aa5dcc-a8c9-4c09-85a4-fe153aec9c61.png)

![image](https://user-images.githubusercontent.com/97100091/225506110-5957f1e8-96c8-444c-bfa6-422c84858de5.png)

### Wireframe :point_down: 
![image](https://user-images.githubusercontent.com/97100091/225506187-d4a45ac7-d116-4918-9c01-2d8584c2ce8b.png)